notebooks/api/0.8/10-container-images.ipynb

```diff
@@ -33,7 +33,47 @@
     "\n",
     "# syft absolute\n",
     "from syft.service.worker.image_registry import SyftImageRegistry\n",
-    "from syft.service.worker.worker_image import SyftWorkerImage"
+    "from syft.service.worker.worker_image import SyftWorkerImage\n",
+    "\n",
+    "# Local registry to test external registry\n",
+    "\n",
+    "\n",
+    "class LocalRegistryContainer:\n",
```

Comment:
It would be good to do the optimization we did in this notebook to also the K8s notebooks in a   later PR.